cli/file_fetcher.rs

```diff
@@ -1666,7 +1684,7 @@ mod tests {
     let (file_fetcher, _) =
       setup(CacheSetting::RespectHeaders, Some(temp_dir.clone()));
     let result = file_fetcher
-      .fetch(&specifier, &mut Permissions::allow_all())
+      .fetch(&specifier, Arc::new(Mutex::new(Permissions::allow_all())))
```

Comment:
Lots of boilerplate. This could be buried in a PermissionsContainer struct-> PermissionsContainer::allow_all()